README.md

```diff
@@ -347,6 +348,43 @@ certutil -hashfile <file> SHA1
 Known good *SHA1* checksum of the Darkflame Universe client:
 - `91498e09b83ce69f46baf9e521d48f23fe502985` (packed client, zip compressed)
 
+# Docker
+
+## Standalone
+
+For standalone deployment, you can use the image provided via Github's Container Repository:
+`ghcr.io/darkflameuniverse/darkfameserver`
+
+This assumes that you have a database deployed to your host or in another docker container.
+
+A basic deployment of this contianer would look like:
+```sh
+# example docker contianer deployment
+docker run -it \
+    -v /path/to/configs/:/app/configs \
+    -v /path/to/logs/:/app/logs \
+    -v /path/to/dumps/:/app/dumps \
+    -v /path/to/res:/app/res:ro \
+    -v /path/to/resServer:/app/resServer \
+    -e DLU_CONFIG_DIR=/app/configs \
+    -e DUMP_FOLDER=/app/dumps \
+    -p 1001:1001/udp \
+    -p 2005:2005/udp \
+    -p 3000-3300:3000-3300/udp \
+ghcr.io/darkflameuniverse/darkfameserver:latest
+```
```

Comment:
imo this should be moved to a shell.example file for people to edit instead in one place to just cut the middle man step of needing to copy this text into a file and instead of copy pasting a file and renaming it.  Plus then we can reference the example file by name and not have a huge docker container command in the readme

Reply:
this is standard to have in the readme for containers

Reply:
Can't say I agree but alright